.circleci/config.yml

```diff
@@ -127,7 +127,7 @@ jobs:
       - run:
           name: run tests
           command: |
-            python -m pytest tests/ -v
+            python -m pytest tests/test_shell.py -v
```

Comment:
Why did you change the tests command here?

Reply:
Just want to check the test_find in test_shell first. So changed it for simplicity. Forgot to revert back